pilot/cmd/pilot-agent/status/util/listeners.go

```diff
@@ -69,6 +70,27 @@ func GetInboundListeningPorts(localHostAddr string, adminPort uint16, nodeType m
 	return ports, buf.String(), nil
 }
 
+// GetVirtualListenerPorts returns ports of Istio's traffic capture listeners.
+func GetVirtualListenerPorts(localHostAddr string, adminPort uint16) ([]uint16, error) {
+	buf, err := doHTTPGet(fmt.Sprintf("http://%s:%d/listeners?format=json", localHostAddr, adminPort))
```

Comment:
Just a sidenode: `/listeners` is also a v2alpha API. See the discussion in https://github.com/istio/istio/issues/14061 and Envoy doc: https://www.envoyproxy.io/docs/envoy/latest/operations/admin